vxhunter/vxutility/symbol_table.py

```diff
@@ -110,56 +104,47 @@ def parse_sym(offset, sym_st_fmt, sym_size, sym_types):
     return sym_dict(offset, *fields), is_sym_valid(sym_types, *fields)
 
 
-def get_symtab_bounds(blk, vx_ver, symbol_fn):
-    '''
-    Try to get the VxWorks 5, 6, or 7 symbol table.
-    '''
+def fix_image_base(blk, vx_ver):
+    """Note: this may seem redundant to duplicate effort, trying to find symtable here and then again
+    in get_symtab_bounds(). However rebasing is needed
+    """
     sym_st_fmt, sym_st_size = get_symbol_fmt(endy_str, word_str, vx_ver)
     sym_types = SYM_TYPES[vx_ver]
 
-    symtab = []
-    i = blk.start.offset
-
-    while i < blk.end.offset - sym_st_size:
-        sym, sym_valid = parse_sym(i, sym_st_fmt, sym_st_size, sym_types)
-
-        # if the current symbol struct is not a valid symbol, only increment the cursor by `word_size` bytes
-        # in case we are unaligned with the symbol table (this assumes the symbol table is word-aligned)
-        if sym is None or not sym_valid:
-            symtab = []
-            i += word_size
-            continue
-
-        name_addr = sym['name_addr']
+    left_off, symtab = find_sym_table(blk, blk.start.offset, sym_st_fmt, sym_st_size, sym_types, vx_ver)
```

Comment:
It looks like `left_off` isn't used. Is it used somewhere else?

Reply:
Correct, I can `_` these instances of this.